app/controllers/check_ins_controller.rb

```diff
@@ -25,10 +25,10 @@ def create
   end
 
   private def merge_worked_at_and_account_into_log_entries(check_in_params)
-    worked_at = check_in_params.delete(:worked_at)
+    worked_at = check_in_params[:worked_at]
     check_in_params[:log_entries_attributes].each do |_id, log_entry|
-      log_entry[:worked_at] = worked_at
       log_entry[:account] = current_account
+      log_entry[:worked_at] = worked_at
```

Comment:
Could we [inline the `worked_at` variable](https://www.lynda.com/Developer-Programming-Foundations-tutorials/Inline-Method-refactoring/122457/135629-4.html?bm=1) since the assignemnt is only used once?